packages/cli/templates/cloudflare/vite.config.ts

```diff
@@ -5,6 +5,10 @@ import {
 import { defineConfig } from "vite";
 import tsconfigPaths from "vite-tsconfig-paths";
 
-export default defineConfig({
-  plugins: [remixCloudflareDevProxy(), remix(), tsconfigPaths()],
-});
+export default defineConfig(({ mode }) => ({
+  plugins: [
+    mode === "production" ? undefined : remixCloudflareDevProxy(),
```

Comment:
remix doesn't handle it under the hood?

Reply:
Added comment, bug in remix proxy
see `//without this, remixCloudflareDevProxy trying to load workerd even for production (it's not needed for production)`